docs/private/BUILD

```diff
@@ -1 +1,5 @@
-# No targets in this package
+# This package only contains source targets
+
+package(default_applicable_licenses = ["//:license"])
+
+licenses(["notice"])
```

Comment:
We don't need licenses notice.  Only google uses it and it doesn't mean anything there any more.

Reply:
The internal licenserules doc currently says "The transition to the new license() rules is not yet complete, so both kinds of declarations are still necessary." :/

Reply:
No one is updating that doc.  It only matters if it is licensees(restricted)

Reply:
Interesting! In that case, let's remove `licenses()`.